homeassistant/components/climate/melissa.py

```diff
@@ -123,6 +124,13 @@ def state(self):
             return self.melissa_state_to_hass(
                 self._cur_settings[self._api.STATE])
 
+    @property
+    def device_state_attributes(self):
+        """Add the attributes that's available."""
+        data = self._data.copy()
```

Comment:
We shouldn't just expose all the data because we can. Please explicitly write out which attributes you want to make part of the state and don't use `copy`.